ship.config.js

```diff
@@ -6,5 +9,13 @@ module.exports = {
     }
   },
   buildCommand: () => "yarn build && /bin/bash ./pre-deploy.sh",
-  pullRequestReviewer: "@algolia/instantsearch-for-websites"
+  pullRequestReviewer: ["@algolia/instantsearch-for-websites"],
+  versionUpdated: ({ version, dir }) => {
+    const readmePath = path.resolve(dir, "README.md");
+    let content = fs.readFileSync(readmePath).toString();
+    const regex = /cdn\.jsdelivr\.net\/npm\/search-insights@(\d+?\.\d+?\.\d+?)/;
```

Comment:
could the version not also have text (like `alpha`)?

Reply:
It could, but we probably won't mention installing an `alpha` version from the README.